docker-compose.ac.yml

```diff
@@ -54,5 +54,29 @@ services:
     logging:
       options: { max-file: "5", max-size: 100m }
     user: "2000:2000"
-
+  ms-ledger:
+    image: "automated-checkout/ms-ledger:dev"
+    container_name: automated-checkout_ms-ledger
+    depends_on:
+      data:
+        condition: service_started
+      command:
+        condition: service_started
+    environment:
+      EDGEX_SECURITY_SECRET_STORE: "false"
+      REGISTRY_HOST: edgex-core-consul
+      SERVICE_HOST: ms-ledger
+      APPCUSTOM_INVENTORYENDPOINT: "http://ms-inventory:48095/inventory"
+    hostname: ms-ledger
+    networks:
+      edgex-network: {}
+    ports:
+    - 127.0.0.1:48093:48093/tcp
+    restart: always
+    ipc: none
+    security_opt:
```

Comment:
Missing `read_only: true` here and in `ms-inventory`

Reply:
Doc: https://intel-iot-devkit.github.io/automated-checkout/automated-checkout-services/micro_services.html#inventory-service, there is post to inventory which will write new items into file. I see the same for ms-ledger, so I remove the read_only: true

Reply:
you can add docker volume to it and still use read only: true.

Reply:
with docker volume app can write file to it assuming the owner and file permission is allowed.

Reply:
The issue is the json files are premade and being copied directly into the docker image. Moving to a docker volume is definitely the way to go but would require some code refactoring which I think would require a new PR. To keep the PR in scope I would recommend creating a new issue on the repo for moving the json files into the volumes and add it to our tech debt story.

Reply:
since current task is for upgrading only, I have created an issue: https://github.com/intel-iot-devkit/automated-checkout/issues/12 to work on moving json files into volumes later.

Reply:
Volume can still be used for that location where the file is in the docker image.

Reply:
is this resolved yet?

Reply:
issue: https://github.com/intel-iot-devkit/automated-checkout/issues/12 created to work on later